Source/Parser/JsonLDParser.swift

```diff
@@ -160,8 +154,7 @@ class JsonLDParser {
 			things[thing.id] = thing
 		}
 		else {
-			let (parsedAttributes, parsedEmbbededThings) = parseAttributes(json: value, 
-																		   context: context)
+			let (parsedAttributes, parsedEmbbededThings) = parseAttributes(json: value, context: context)
```

Comment:
While we don't have a defined SF, it may be easier to keep as it is for now. Can you revert this hunk?